theme.ts

```diff
@@ -12,3 +15,23 @@ export const colors = {
   black: "#444444",
   focused: "#2D9CDB"
 };
+
+export const useLoadFonts = (props: {
+  setFontLoaded: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
You can use hooks inside custom hooks, so you can just return the boolean state from this hook, eg.
```
export const useCustomFonts = () => {
  const [fontLoaded, setFontLoaded] = useState(false);
  useEffect(() => {
    loadFonts()
      .then(() => setFontLoaded(true))
      .catch(() => /* TODO */);
  }, []);
  return fontLoaded;
})
```

btw, I would move the `async function loadFonts` declaration outside the hook, right before the hook declaration is ideal. I do this because I find it easier to understand what the hooks does when the details of the font loading is abstracted away.

Reply:
I find the following:
```
Font.loadAsync({
        "Montserrat-Regular": require("./assets/fonts/Montserrat-Regular.ttf"),
        "Montserrat-SemiBold": require("./assets/fonts/Montserrat-SemiBold.ttf"),
        "Montserrat-Medium": require("./assets/fonts/Montserrat-Medium.ttf"),
        "Montserrat-Bold": require("./assets/fonts/Montserrat-Bold.ttf")
      })
```
to be mostly noise, and does not help me understand what the hook accomplishes.

Reply:
fixed